packages/preview/slydst/0.1.0/README.md

```diff
@@ -0,0 +1,82 @@
+# Slydst
+
+Create simple static slides with Typst.
+
+Slydst allows the creation of slides using Typst headings.
+This simplicity comes at the expense of dynamic content such as subslide animations.
+For more complete and complex slides functionalities, see other tools such as Polylux.
+
+## Usage
+
+To start, just use the following preamble (only the title is required).
+
+```typst
+#import "@preview/slydst:0.1.0": *
+
+#show slides.with(
+    title: "Insert your title here", // Required
```

Comment:
Two spaces indent would be more idiomatic for Typst, in case you want to change it.

Reply:
Indeed, I have used two spaces indents in `README.md` and all source files `lib.typ`, `slydst.typ`,  and `example.typ`.